install.sh

```diff
@@ -0,0 +1,4355 @@
+    --vsphere-vcenter-endpoint)         shift
+                                        VSPHERE_VCENTER=$1
+                                        ;;
+    --vsphere-vcenter-port)             shift
+                                        VSPHERE_VCENTER_PORT=$1
+                                        ;;
+    --vsphere-vcenter-datastore-prefix) shift
+                                        VSPHERE_DATASTORE_PREFIX=$1
+                                        ;;
+    --vsphere-vcenter-install-mode)     shift
+                                        VSPHERE_INSTALL_MODE=$1
+                                        ;;
+    --vsphere-user)                     shift
+                                        VSPHERE_USER=$1
+                                        ;;
+    --vsphere-password)                 shift
+                                        VSPHERE_PASSWORD=$1
+                                        ;;
+    --domain)                           shift
+                                        DOMAIN=$1
+                                        ;;
+    --ingress-controller)                   INGRESS_CONTROLLER_PROVISION="true"
+                                            ;;
+    --re-configure)                         RE_DEPLOY_ON_SAME_CLUSTER="true"
+                                            ;;
+    --portworx-disk-provision-secret-name)  shift 
+                                            PX_DISK_PROVISION_SECRET_NAME=$1
+                                            ;;
+    --proxy-forwarding)                     ENABLE_PROXY_FORWARDING="true"
+                                            ;;
+    --proxy-forwarding-uat)                 ENABLE_PROXY_FORWARDING_FOR_UAT="true"
+                                            ;;
+    --log-location)                         shift
+                                            PXCENTRAL_INSTALL_SCRIPT_LOG_LOCATION=$1
+                                            ;;
+    -h | --help )   usage
+                    ;;
+    * )             usage
+    esac
+    shift
+done
+
+TIMEOUT=1800
+SLEEPINTERVAL=2
+LBSERVICETIMEOUT=300
+PXCNAMESPACE_DEFAULT="portworx"
+
+UATLICENCETYPE="false"
+AIRGAPPEDLICENSETYPE="false"
+ISOPENSHIFTCLUSTER="false"
+OPERATOR_UNSUPPORTED_CLUSTER="false"
+ISCLOUDDEPLOYMENT="false"
+PXCPROVISIONEDOIDC="false"
+CLOUDSTORAGEENABLED="false"
+AWS_CLOUD_PLATFORM="false"
+AZURE_CLOUD_PLATFORM="false"
+GOOGLE_CLOUD_PLATFORM="false"
+IBM_CLOUD_PLATFORM="false"
+
+EKS_CLUSTER_TYPE="false"
+GKE_CLUSTER_TYPE="false"
+AKS_CLUSTER_TYPE="false"
+
+AWS_DISK_TYPE="gp2"
+GCP_DISK_TYPE="pd-standard"
+AZURE_DISK_TYPE="Premium_LRS"
+DEFAULT_DISK_SIZE="150"
+
+PXENDPOINT=""
+maxRetry=5
+
+ONPREMOPERATORIMAGE="portworx/pxcentral-onprem-operator:1.0.3"
+PXCENTRALAPISERVER="portworx/pxcentral-onprem-api:1.0.3"
+PXOPERATORIMAGE="portworx/px-operator:1.3.1"
+PXCPRESETUPIMAGE="portworx/pxcentral-onprem-pre-setup:1.0.1"
+PXDEVIMAGE="portworx/px-dev:2.5.0"
+STORK_IMAGE="openstorage/stork:2.4.0"
```

Comment:
Should the image be 2.4.1?

Reply:
Currently, we are using 2.4.0 will change this in 1.0.4 release.